maltoolbox/language/languagegraph.py

```diff
@@ -26,44 +26,58 @@
 
 logger = logging.getLogger(__name__)
 
+def get_asset_name_from_full_attack_step(
+        attack_step_full_name: str) -> str:
+    return attack_step_full_name.split(':')[0]
+
+def get_attack_step_name_from_full_attack_step(
+        attack_step_full_name: str) -> str:
+    return attack_step_full_name.split(':')[1]
```

Comment:
These two are always used in pair. I suggest:

```suggestion
def decompose_full_name(
        full_name: str) -> list[str]:
    return attack_step_full_name.split(':')
```

And use it like:

```
target_asset_name, target_attack_step_name = decompose_attack_step_full_name(target_full_attack_step_name)
```